### ISSUE TITLE
figcaption reads when user tabs to show/hide button

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/aria-practices#693]
https://www.w3.org/TR/wai-aria-practices-1.1/examples/disclosure/disclosure-img-long-description.html

When tabbing to the button "Data table for Minard's Chart" ...  using JAWS or NVDA

![image](https://user-images.githubusercontent.com/2365411/40446695-18b81434-5e9e-11e8-984d-9d58bd35a82a.png)
 which is to expand the long description the following reads:

"Figurative Map of the successive losses in men of the French Army in the Russian campaign 1812-1813. Drawn by Mr. Minard, Inspector General of Bridges and Roads in retirement. Paris, 20 November 1869. The numbers of men present are represented by the widths of the colored zones in a rate of one millimeter for ten thousand men; ... ."

 I'm guessing JAWS and NVDA are reading the figcaption when the button is focused.

That's a very long button name.

**Comment by jnurthen** [comment]
@asurkov I've got no idea why JAWS with FF is not reading the button text when navigating to it. Any chance you let us know where you think the blame lies (either to the example, FF or the AT)

**Comment by DavidMacDonald** [comment]
OK, will investigate, BTW it was Chrome not FF

**Comment by DavidMacDonald** [comment]
If an example doesn't work in JAWs or NVDA properly, even if its because of bugs in those technologies, should it be called a best practice?
